cli/src/factory_ops/unboxing/init

```diff
@@ -367,6 +367,18 @@ echo "Ok." | tee -a ${LOG_FILE}
 # lock unboxing user
 lock_unboxing_user
 
+# FIXME the value might be encrypted
+service_user=$(salt srvnode-2 pillar.get system:service_user:name --output=newline_values_only)
+if [[ -n "$service_user" ]]; then
+    echo -n "Activating service user $service_user on the secondary node................." | tee -a ${LOG_FILE}
+    salt "srvnode-2" state.single user.present name="$service_user" expire=-1 ${salt_opts}
```

Comment:
why are we not doing this on node-1?

Reply:
It is activated there already by [design](https://jts.seagate.com/browse/EOS-15007?focusedCommentId=1976996&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1976996)

Reply:
ok!